scripts/testdata/satellite-config.yaml.lock

```diff
@@ -10,6 +10,12 @@
 # how frequently checker should audit segments
 # checker.interval: 30s
 
+# how frequently irrepair checker should audit segments
+# checker.irrepair-interval: 15s
+
+# maximum segments that can be repaired concurrently
```

Comment:
Use a name that reflects that its checker and irreparable, rather than repair

Reply:
Changed to IrrdbInterval